typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -86,10 +165,7 @@ const hyperlane: RootAgentConfig = {
 const releaseCandidate: RootAgentConfig = {
   ...contextBase,
   context: Contexts.ReleaseCandidate,
-  contextChainNames: {
-    ...contextBase.contextChainNames,
-    [Role.Validator]: ethereumChainNames,
-  },
+  contextChainNames: hyperlaneContextAgentChainNames,
```

Comment:
When these need to diverge, will we just destructure `hyperlaneContextAgentChainNames` and overwrite here? So to roll out a new chain we'd add a new `false` entry to the hyperlane context and then overwrite to true in `rc`?

Reply:
That works, I have less of a strong opinion about how non-hyperlane contexts specify things. My main goal was to just make sure that the hyperlane context isn't reliant on configurations done for managing contracts

But my general thoughts are:
- hyperlane & rc should always have the same chains enabled
- if RC deviated from this, either another `AgentChainConfig` written out or the destructure & overwrite may be natural, I'm unsure if I have an opinion
- any other contexts should specify chains explicitly, like how we do for the neutron context